app/scripts/templates/settings/two_step_authentication.mustache

```diff
@@ -59,7 +59,7 @@
         <form novalidate>
             <div id="totp" class="totp-details hidden">
                 <div class="qr-image-container">
-                    <img class="qr-image"/>
+                    <img class="qr-image" alt="{{#t}}Two step authentication QR code image{{/t}}"/>
```

Comment:
You could also use the secret as the alt tag, at a minimum screen readers wouldn't have to click the "Can't scan the code" link.

BTW, the `show-code-link` doesn't have an `href` either.